packages/client/ui/react-ui/src/providers/auth/web3/WagmiAuthProvider.tsx

```diff
@@ -0,0 +1,25 @@
+import type React from "react";
+import { createConfig, http, WagmiProvider } from "wagmi";
+import { mainnet, sepolia } from "wagmi/chains";
+// todo: investigate implications of using react-query
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import { metaMask, walletConnect } from "wagmi/connectors";
+
+const config = createConfig({
+    chains: [mainnet, sepolia],
```

Comment:
can this be dependent on the chain they set in the auth provider?

Reply:
I don't think so at least not in the current state as we're only accepting a `defaultChain` prop

Reply:
then again, perhaps we can